receiver/ciscoosreceiver/README.md

```diff
@@ -0,0 +1,96 @@
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/component-stability.md#development
+<!-- end autogenerated section -->
+
+The Cisco OS Receiver is a modular receiver that collects metrics from Cisco network devices via SSH connections. It supports multiple scrapers for comprehensive network monitoring and provides native OpenTelemetry OTLP metrics generation.
+
+## Configuration
+
+The following settings are available:
+
+| Setting | Type | Required | Description |
+|---------|------|----------|-------------|
+| `devices` | []DeviceConfig | Yes | List of Cisco devices to monitor |
+| `collection_interval` | duration | No | How often to collect metrics (default: 60s) |
+| `timeout` | duration | No | SSH connection and command timeout (default: 30s) |
+| `scrapers` | ScrapersConfig | Yes | Enable/disable specific scrapers |
+
+### Device Configuration
+
+| Setting | Type | Required | Description |
+|---------|------|----------|-------------|
+| `host` | string | Yes | Device address in `host:port` format |
+| `username` | string | Yes | SSH username for authentication |
+| `password` | string | No* | Password for authentication |
+| `key_file` | string | No* | Path to SSH private key file |
+
+*Either `password` or `key_file` is required, but not both.
+
+### Scrapers Configuration
+
+| Setting | Type | Description |
+|---------|------|-------------|
+| `bgp` | bool | BGP session metrics |
+| `environment` | bool | Temperature and power metrics |
+| `facts` | bool | System information metrics |
+| `interfaces` | bool | Interface status and statistics |
+| `optics` | bool | Optical transceiver metrics |
+
+## Scrapers
+
+The receiver supports the following scrapers:
+
+- **BGP**: Collects BGP session information and statistics
+- **Environment**: Collects temperature and power consumption metrics  
+- **Facts**: Collects system information (OS version, memory, CPU utilization)
+- **Interfaces**: Collects interface statistics (bytes, packets, errors)
+- **Optics**: Collects optical signal strength information
+
+## Example Configuration
+
+```yaml
+receivers:
+  ciscoosreceiver:
+    collection_interval: 60s
+    timeout: 30s
+    scrapers:
+      bgp: true
+      environment: true
+      facts: true
+      interfaces: true
+      optics: true
+    devices:
+      - host: "cisco-device:22"
+        username: "admin"
+        password: "password"
+
+exporters:
+  debug:
+    verbosity: detailed
+
+service:
+  pipelines:
+    metrics:
+      receivers: [ciscoosreceiver]
+      exporters: [debug]
+```
+
+## Metrics
+
+| Metric | Type | Description | Attributes |
+|--------|------|-------------|------------|
+| `cisco_up` | Gauge | Device connectivity status (1=connected, 0=disconnected) | `target` |
+
+## Development Status
+
+This component is currently in **development** status. It is not recommended for production use.
```

Comment:
there is no real need to make the point twice, as this is already communicated in the status box up there. This is mostly a nit but we find this type of warning gets forgotten behind well after the component has matured, scaring users. You might like to remove it, but it's not a blocker.